packages/medusa/src/api/routes/admin/variants/list-variants.ts

```diff
@@ -149,6 +149,7 @@ export default async (req, res) => {
     currency_code: currencyCode,
     customer_id: req.validatedQuery.customer_id,
     include_discount_prices: true,
+    ignore_cache: true,
```

Comment:
self-review: now that we invalidate the cache in the service method, `ignore_cache` isn't necessary anymore since we will always have the most recent prices on Admin, so we technically don't need the flag but it might be useful to keep an option to force calculation if needed, wdyt?